CHANGELOG.md

```diff
@@ -8,14 +8,14 @@
 * Logging duration (etcd connection establishment, kafka connection establishment, resync)
 
 ## Plugin Configuration
-* new [examples/plugin_config](examples/plugin_config)
+* new [examples/plugin_config](https://github.com/ligato/cn-infra/blob/master/config/plugin_config.go)
```

Comment:
Why do we need absolute URLs here? Absolute URLs would make sense if there would be a particular commit as encoded in URL.

Reply:
done